packages/editor/src/components/block-toolbar/style.scss

```diff
@@ -1,5 +1,5 @@
 .editor-block-toolbar {
-	display: inline-flex;
+	display: flex;
```

Comment:
This change broke the toolbar on wide images. It should be this:

![screen shot 2018-08-17 at 13 34 56](https://user-images.githubusercontent.com/1204802/44264382-ab389500-a222-11e8-9139-db78632efaf0.png)

But with flex instead of inline flex, it's now this:

![screen shot 2018-08-17 at 13 35 39](https://user-images.githubusercontent.com/1204802/44264393-b4296680-a222-11e8-9b74-f5ad845ad3ee.png)